packages/odyssey-react/src/components/TextArea/TextArea.tsx

```diff
@@ -12,18 +12,18 @@
 
 import React, { useCallback } from "react";
 import type {
-  ComponentProps,
   FunctionComponent,
   FocusEventHandler,
   ChangeEvent,
   RefCallback,
 } from "react";
-import styles from "./TextArea.module.scss";
-import { useOid, useCx, withStyles } from "../../utils";
+import { withTheme } from "@okta/odyssey-react-theme";
+import { useOid, useCx } from "../../utils";
 import { Field } from "../Field";
 import type { SharedFieldTypes } from "../Field/types";
+import styles from "./TextArea.module.scss";
 
-interface Props extends SharedFieldTypes {
+export interface TextAreaProps extends SharedFieldTypes {
```

Comment:
Is this a new convention? `${ComponentName}Props`

Would be good to doc this kind of stuff

Reply:
@arnoldsandoval yea since we want to share props as part of our top level API just prefixing the component name here makes using barrel file exports elsewhere easier. We've been doing this for a month or so. We should absolutely write up some component guidelines information for our doc site and cover topics like this